sklearn/linear_model/tests/test_huber.py

```diff
@@ -0,0 +1,180 @@
+# Authors: Manoj Kumar mks542@nyu.edu
+# License: BSD 3 clause
+
+import numpy as np
+from scipy import optimize, sparse
+
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_almost_equal
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_greater
+
+from sklearn.datasets import make_regression
+from sklearn.linear_model import HuberRegressor, Ridge, SGDRegressor
+from sklearn.linear_model.huber import _huber_loss_and_gradient
+
+rng = np.random.RandomState(0)
+
+
+def make_regression_with_outliers(n_samples=50, n_features=20):
+    X, y = make_regression(
+        n_samples=n_samples, n_features=n_features,
+        random_state=0, noise=10.0)
+
+    # Replace 10% of the sample with noise.
+    num_noise = int(0.1 * n_samples)
+    random_samples = rng.randint(0, n_samples, num_noise)
+    X[random_samples, :] = rng.normal(0, 1, (num_noise, X.shape[1]))
+    return X, y
+
+
+def test_huber_equals_ridge_for_high_epsilon():
+    """Test that Ridge matches HuberRegressor for large epsilon"""
+    X, y = make_regression_with_outliers()
+    ridge = Ridge(fit_intercept=True, alpha=0.0, solver='svd')
+    ridge.fit(X, y)
+    huber = HuberRegressor(fit_intercept=True, epsilon=1e2, alpha=0.0)
+    huber.fit(X, y)
+    assert_almost_equal(huber.coef_, ridge.coef_, 2)
```

Comment:
why only 2? Are the solvers struggling because of alpha=0? Shouldn't we use ``LinearRegression`` instead of ``Ridge`` then?